_includes/scripts.html

```diff
@@ -12,13 +12,116 @@
 {% if site.ga_tracker %}
 
 <script>
-  window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments) }; ga.l = +new Date;
-  ga('create', '{{ site.ga_tracker }}', 'auto');
-  ga('require', 'eventTracker');
-  ga('require', 'maxScrollTracker');
-  ga('require', 'outboundFormTracker');
-  ga('require', 'outboundLinkTracker');
-  ga('send', 'pageview');
+  window.ga =
+    window.ga ||
+    function() {
+      (ga.q = ga.q || []).push(arguments);
+    };
+  ga.l = +new Date();
+
+  ga("create", "{{ site.ga_tracker }}", "auto");
+  ga("require", "eventTracker");
+  ga("require", "maxScrollTracker");
+  ga("require", "outboundFormTracker");
+  ga("require", "outboundLinkTracker");
+  ga("send", "pageview");
+
+  // Google Analytics Event Tracking for IAC Page A/B Tests
+  const IACVersion = getCookiebyName("IACLibraryVersion");
+
+  const createEventAction = (tag) => {
+    return IACVersion ? `${tag} (${IACVersion})` : tag;
+  }
+
+  const gaEventBaseObject = {
+    hitType: "event",
+    eventCategory: "CTA Clicks",
+    eventValue: 1
+  }
+
+  $("a").on("click", function() {
+    const href = $(this).attr("href");
+    if (!href) return;
+
+    if (href.includes("/contact")) {
```

Comment:
Could prob define these click handlers more declaratively:

```js
const gaEvents = [
  {urlIncludes: "/contact", tag = "Click Contact Us", label: "User went to Contact Page"},
  {urlIncludes: "/pricing", tag = "Click Buy Now", label: "User went to Pricing Page"},
  // ... etc ...
]
```

Reply:
Also... These don't capture events on the SaaS app pages (`/infrastructure-as-code-library`). as that loads different scripts. So wouldn't this only do tracking for the old IaC Lib design?